planter/README.md

```diff
@@ -1,4 +1,7 @@
-# Planter
+# Planter 
+Bazel in a container.
+
+<img src="planter-logo.svg" />
```

Comment:
Nice logo :dancer:

Reply:
now I want a shipping container to use as a gardening planter.